notebooks/203-meter-reader/203-meter-reader.ipynb

```diff
@@ -548,7 +536,9 @@
     "det_model_path = f\"{MODEL_DIR}/meter_det_model/model.pdmodel\"\n",
     "det_model_shape = {'image': [1, 3, 608, 608], 'im_shape': [1, 2], 'scale_factor': [1, 2]}\n",
     "seg_model_path = f\"{MODEL_DIR}/meter_seg_model/model.pdmodel\"\n",
-    "seg_model_shape = {'image': [-1, 3, 512, 512]}\n",
+    "seg_model_shape = {'image': [ov.Dimension(1, 2), 3, 512, 512]}\n",
+    "# seg_model_shape = {'image': [-1, 3, 512, 512]}\n",
```

Comment:
Please remove that comment or add an explanation that this is an alternative. Commented code is very confusing. No one knows if it wasn't removed by mistake or if it was intentional.